src/Mock.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Mockman from "mockman-js";
+
+function Mock() {
+  return (
+    <div className="MockAPI">
+      <Mockman />
+    </div>
+  );
+}
+
```

Comment:
you have 2 similar components ( src/mock & component/mock ) with the same functionality remove any one of them